tracer/src/Datadog.Trace/Telemetry/Collectors/ConfigurationTelemetry.Collector.cs

```diff
@@ -1,4 +1,4 @@
-// <copyright file="ConfigurationTelemetryCollectorV2.cs" company="Datadog">
+// <copyright file="ConfigurationTelemetry.Collector.cs" company="Datadog">
```

Comment:
I find the new name weird. First a dot... Why? But mainly, why `ConfigurationTelemetry.Collector` whereas we have `MetricsTelemetryCollector`. That doesn't feel consistent.

That said I don't mind much

Reply:
Blame @kevingosse 😉 

The reason is that this class contains the `partial` `ConfigurationTelemetry` class. It's the "collector" part of the implementation. Mostly it's an artifact of the fact I changed the whole design part way through, and didn't want to rename `ConfigurationTelemetry` and have to touch every file in the codebase 😬

Reply:
Thanks for the explanation. As I said I don't mind much and I'm happy to blame @kevingosse.